### ISSUE TITLE
Fix Query headings that require a design solution

### ISSUE THREAD

**Issue by eelzi-ONRR** [DOI-ONRR/nrrd#3526]
On Query page - Query Data and Download tabs look the same - one is an H1 and the other is not. Headings that look the same but are not coded the same is a WCAG violation.

**Comment by lpgoldstein** [comment]
Working on other design issues that will get rid of this issue.

**Comment by lpgoldstein** [comment]
No longer relevant
